aws/request/handlers.go

```diff
@@ -20,22 +20,6 @@ type Handlers struct {
 	AfterRetry       HandlerList
 }
 
-// Copy returns of this handler's lists.
-func (h *Handlers) Copy() Handlers {
```

Comment:
Users are always free to add or remove request handlers from the various HandlerLists based on  their use cases. This can be performed both on the service client instance, and the individual request such as with, [`dynamodb.CreateTableRequest`](http://docs.aws.amazon.com/sdk-for-go/api/service/dynamodb/#DynamoDB.CreateTableRequest). All of the xxxRequest methods return a `request.Request` that can be modified based on the user's preferences. This is commonly done for debugging, or metrics reporting.

In addition there are customizations the SDK implements for some of the service client's and their API operations, and these are applied to the individual operation requests as they are called.

The HandlerList could be created once the first time each API operation is called by the service client, but again mutex's would need to be added around the HandlerLists because API operations can be called concurrently. Though again, the handler lists must be a unique copy because users could choose to augment an operation's request Handlers uniformly. An upfront, or first time use copy would prevent this functionality.